### DISCUSSION TITLE
Support for SPF Macros

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#2413] [Feature suggestions]
Would be amazing to add support for SPF Macros https://www.jamieweb.net/blog/using-spf-macros-to-solve-the-operational-challenges-of-spf/

**Comment by willpower232** [comment]
I had never heard of SPF macros, they sound quite powerful but I can imagine the code to process them is quite complicated :sweat_smile: 

Thankfully Postal still sends the emails regardless of DNS validation so as long as you're confident in whatever arrangement of SPF macros you have, then it will be fine for now until someone figures it out.

**Reply by uhlhosting** [reply]
@willpower232 I believe only the fact that they are so low in use, and yet so powerful make me think will take some time to get them implemented, yet they do seem powerful and there will be only benefits implementing them.
